design/FY2016/NFP-AFNPolygonalWindows.md

```diff
@@ -0,0 +1,491 @@
+
+\subsubsection{Field: Venting Availability Schedule Name}\label{field-venting-availability-schedule-name-1}
+
+The name of a schedule that specifies when venting is available. A zero or negative schedule value means venting is not allowed. A value greater than zero means venting can occur if other venting control conditions (specified by Ventilation Control Mode and Vent Temperature Schedule Name) are satisfied. This schedule name should not be confused with Vent Temperature Schedule Name.
+
+If a Venting Availability Schedule Name is not specified, it is assumed that venting is always available.
+
+Using Venting Availability Schedule allows you to turn off venting at certain times of the day (at night, for example), week (on weekends, for example), or year (during the winter, for example).
+
+If used with Ventilation Control Mode = Constant, the ventilation rate is constant only when this schedule allows venting; otherwise the ventilation rate is set to zero.
+
+If Ventilation Control Mode = NoVent, this schedule has no effect.
+
+\textbf{Note:} In order to establish an airflow network, each AirflowNetwork:MultiZone:Zone object must have at least two surfaces defined with AirflowNetwork:MultiZone:Surface objects, so that air can flow from one zone into other zones (or to outdoors) through the network (air mass flow conserved). In addition, for all AirflowNetwork:MultiZone:Surface objects facing the same Zone Name (ref. BuildingSurface:Detailed), at least two different environments must be defined for the other side of these surfaces (e.g., an external node and an adjacent zone, two adjacent zones, or two external nodes).
+
+\subsubsection{Field: Occupant Ventilation Control Name}\label{field-occupant-ventilation-control-name-1}
+
+The name of an AirlowNetwork:OccupantVentilationControl object. The object is used to perform advanced window opening control based on occupant conditions. When an object name is given, advanced window opening control is performed and, the Ventilation Control defined in the Ventilation Control Mode field will be overridden.
+
+**<span style="color:red;">
+\subsubsection{Field: Equivalent Rectangular Shape Choice}\label{equivalent-rectangular-shape- choice}bold red</span>**
+
+**<span style="color:red;">
+This field is applied to a non-rectangular window or door. The equivalent surface has the same area as the non-rectangular one. When Height is entered, the equivalent width is equal to the area divided by the height. When "Aspect Ratio from Input" is entered, the equivalent height is equal to Square Root of Area divided by AspectRatio. The equivalent width is equal to the equivalent height * AspectRatio. When "Parent Surface Aspect Ratio" is entered, the equivalent height is equal to Square Root of Area divided by the parent surface AspectRatio. The equivalent width is equal to the equivalent height * parant surface AspectRatio. </span>**
```

Comment:
@jasondegraw You are right. It should be from an additional field. I am going to add an optional field with default value at 1.